src/Raven.Server/Platform/Posix/MemInfoReader.cs

```diff
@@ -292,6 +292,9 @@ public MemInfo()
         [SnmpIndex(58)]
         public Size Unaccepted { get; set; }
 
+        [SnmpIndex(59)]
+        public Size ShadowCallStack { get; set; }
```

Comment:
This seems unrelated?

Reply:
after switching runners the test for parsing meminfo live fails on Ubuntu-24.04-arm